### ISSUE TITLE
Should be able to accept voiceover with no text

### ISSUE THREAD

**Issue by whitman** [authorly/interapptive-web#985]
Currently text widgets are required for accepting a voiceover in the Voiceover modal. Users should be able to open modal, select a sound, then Accept to close the modal without any text present.
